src/views/Home/Home.vue

```diff
@@ -1,16 +1,18 @@
 <template>
-    <div class="home" id="homePage">
-        <h1>Home Page</h1>
+    <div class="home container-fluid" id="homePage">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Attribute "id" should go before "class".](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)